apis/r/R/TileDBArray.R

```diff
@@ -140,6 +140,71 @@ TileDBArray <- R6::R6Class(
       ))
     },
 
+    used_shape = function(simplify = FALSE, index1 = FALSE) {
```

Comment:
needs a docstring -- this is user-facing API

Reply:
Added in 02fe817